restcomm/restcomm.commons/src/main/java/org/restcomm/connect/commons/util/DigestAuthentication.java

```diff
@@ -77,24 +77,24 @@ public static String response(final String algorithm, final String user, final S
      * @param qop
      * @return
      */
-    public static String response(final String algorithm, final String user, final String realm, final String password,
-            String password2, final String nonce, final String nc, final String cnonce, final String method, final String uri,
+    public static String response(final String authHeaderAlgorithm, final String user, final String realm, final String password,
+            final String clientPasswordAlgorithm, final String nonce, final String nc, final String cnonce, final String method, final String uri,
             String body, final String qop) {
-        validate(user, realm, password, nonce, method, uri, algorithm);
+        validate(user, realm, password, nonce, method, uri, authHeaderAlgorithm);
         String ha1;
 
-        if(!password2.isEmpty()){
-            ha1 = password2;
-        }else{
-            final String a1 = A1(algorithm, user, realm, password, nonce, cnonce);
-            ha1 = H(a1, algorithm);
+        if (clientPasswordAlgorithm.equalsIgnoreCase(RestcommConfiguration.getInstance().getMain().getClearTextPasswordAlgorithm())) {
```

Comment:
why do we need a configuration setting to set the token for ClearText algorithm?

Reply:
@jaimecasero see my previous comment, this is needed so we use the same string thoughout the project.